Gemfile

```diff
@@ -18,9 +18,11 @@ gem 'dm-validations'
 gem 'foreigner'
 gem 'json'
 gem 'rake'
+gem 'rufus-scheduler'
```

Comment:
Can you add these changed gems to the Gemfile.lock as well? (When you installed them, `bundle` should auto update the lockfile IIRC)